src/nginxconfig/templates/global_sections/tools.vue

```diff
@@ -291,8 +295,8 @@ THE SOFTWARE.
             },
             resetDomains() {
                 this.confirm(
-                    'Reset all domain configs',
-                    'Are you sure you want to reset the configuration of ALL domains?',
+                    i18n.templates.globalSections.tools.removeAllDomainsConfig,
+                    i18n.templates.globalSections.tools.removeAllDomainsConfigBody,
```

Comment:
```suggestion
                    i18n.templates.globalSections.tools.resetAllDomainsConfig,
                    i18n.templates.globalSections.tools.resetAllDomainsConfigBody,
```
Let's use `reset` to be clearer.